tests/behavior/transforms/coercer.toml

```diff
@@ -19,7 +19,7 @@
       float_field = "3.1415926535"
       string_field = -1
       timestamp_field = "2020-02-20T12:34:56.123456789Z"
-      timestamp_field_formatted = "2020-02-20 12:34:56"
+      timestamp_field_formatted = "2020-02-20 12:34:56+0000"
```

Comment:
Added this just for consistency; otherwise if your local time isn't UTC, the test will fail.